app.js

```diff
@@ -9,17 +9,26 @@ const punkAPI = new PunkAPIWrapper();
 
 app.set('view engine', 'hbs');
 app.set('views', path.join(__dirname, 'views'));
-
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
 
-// ...
+hbs.registerPartials("views/partials");
+
 
 // Add the route handlers here:
 
-app.get('/', (req, res) => {
-  res.render('index');
-});
+app.get('/', (req, res) => res.render("index", {title: "Home"}));
+
+// Unsure How to Properly Execute 
+// app.get('/beers', (req, res, next) => {
```

Comment:
Hey Chiara,

Let me share a working example with you. :)

```
app.get('/beers', (req, res, next) => {
  punkAPI
    .getBeers()
    .then((responseFromDB) => {
      res.render('beers/beers.hbs', { beers: responseFromDB });
    })
    .catch((error) => console.log(error));
});
```

In this example a "then" method is used to handle the promise.

Another way of handling a promise is using "async/await". This would look as follows:

```
app.get('/beers', async (req, res, next) => {
    try {
        const beers = await punkAPI.getBeers();
        res.render('beers/beers.hbs', { beers: responseFromDB });
    } catch(err) {
        console.log(error);
    }
});
```

I hope these two examples clarify the different ways of handling promises. Please reach out if you have a question.